### DISCUSSION TITLE
Libfm-Qt - Show the file/dir list before deletion.

### DISCUSSION THREAD

**Discussion by maplee0** [lxqt/lxqt#2287] [Ideas]
It would be nice to get the file/dir list before deletion. I assume it would help just to see what is going to be deleted when user has to select many files and there are hundreds of files in the dir.

**Comment by tsujan** [comment]
> I assume it would help just to see what is going to be deleted when user has to select many files and there are hundreds of files in the dir.

Quite conversely and especially in that case, a list will be useless. The user sees the selected files in the view.
